### PR TITLE
address biopet-609, fix broken links, reorganize layout. 

### PR DESCRIPTION
**Pull request by hailiangmei**

### CODE REVIEW

**Comment by hailiangmei** [comment]
deadlink removed. @ffinfo can we merge now?

File: docs/pipelines/shiva.md [hunk 1]
```diff
@@ -2,92 +2,87 @@
...
+* [VEP annotation](toucan.md)
+* [CNV analysis](kopisu.md)
 * <a href="http://broadinstitute.github.io/picard/" target="_blank">Picard tool suite</a>
-* [Flexiprep](flexiprep.md)
 * <a href="https://www.broadinstitute.org/gatk/" target="_blank">GATK tools</a>:
-    * GATK
-    * Freebayes
-    * Bcftools
-    * Samtools
+* Freebayes
```
Comment (ffinfo) [inline docs/pipelines/shiva.md]: Adding link to the websites of the tools?

File: docs/pipelines/shiva.md [hunk 2]
```diff
@@ -2,92 +2,87 @@
...
-biopet pipeline shiva -config MySamples.json -config MySettings.json -run
+biopet pipeline shiva -config MySamples.yml -config MySettings.yml -run
 ~~~
 
 A dry run can be performed by simply removing the `-run` flag from the command line call.
 
-[Gears](gears) is run automatically for the data analysed with `Shiva`. There are two levels on which this can be done and this should be specified in the [config](../general/config) file:
+[Gears](gears.md) is run automatically for the data analysed with `Shiva`. There are two levels on which this can be done and this should be specified in the [config](../general/config) file:
 
 *`mapping_to_gears: unmapped` : Unmapped reads after alignment. (default)
```
Comment (ffinfo) [inline docs/pipelines/shiva.md]: For biopet the default is `none`, on shark the default is `all`

File: docs/pipelines/shiva.md [hunk 3]
```diff
@@ -2,92 +2,87 @@
...
+    - unifiedgenotyper
+    - haplotypecaller_gvcf
+unifiedgenotyper:
+    merge_vcf_results: false # This will do the variantcalling but will not merged into the final vcf file
+```
 ----
 
-## Variant caller
+## Supported variant callers
 At this moment the following variant callers can be used
```
Comment (ffinfo) [inline docs/pipelines/shiva.md:81]: This should be a table with the following items as config value:
- haplotypecaller_gvcf
- haplotypecaller_allele
- unifiedgenotyper_allele
- unifiedgenotyper
- haplotypecaller
- freebayes
- raw
- bcftools
- bcftools_singlesample
- varscan_cns_singlesample

Currently it's unclear what each config value is for each caller

**Review by ffinfo — CHANGES_REQUESTED** [review]
Some comments but can you also remove the yamsvp.md file?

File: docs/pipelines/shiva.md (hunk 1 above)
Comment (hailiangmei) [reply docs/pipelines/shiva.md]: fixed

File: docs/pipelines/shiva.md (hunk 2 above)
Comment (hailiangmei) [reply docs/pipelines/shiva.md]: fixed.

File: docs/pipelines/shiva.md (hunk 3 above)
Comment (hailiangmei) [reply docs/pipelines/shiva.md:81]: add one extra column in the config value table. Can you check?

File: docs/pipelines/shiva.md [hunk 4]
```diff
@@ -104,32 +99,32 @@ At this moment the following variant callers can be used
...
-| vcffilter | min_sample_depth | Integer | 8 | Filter variants with at least x coverage |
-| vcffilter | min_alternate_depth | Integer | 2 | Filter variants with at least x depth on the alternate allele |
-| vcffilter | min_samples_pass | Integer | 1 | Minimum amount of samples which pass custom filter (requires additional flags) |
-| vcffilter | filter_ref_calls | Boolean | true | Remove reference calls |
+| ConfigNamespace | Name |  Type | Default | Function | Applicable variant caller |
+| ----------- | ---- | ----- | ------- | -------- | -------- |
+| shiva | species | String | unknown_species | Name of species, like H.sapiens | all |
+| shiva | reference_name | String | unknown_reference_name | Name of reference, like hg19 | all |
+| shiva | reference_fasta | String |  | reference to align to | all |
+| shiva | dbsnp_vcf | String |  | vcf file of dbsnp records | haplotypecaller, haplotypecaller_gvcf, haplotypecaller_allele, unifiedgenotyper, unifiedgenotyper_allele|
```
Comment (ffinfo) [inline docs/pipelines/shiva.md]: I think this is the wrong line ;)

Comment (hailiangmei) [reply docs/pipelines/shiva.md]: can you correct it?

**Review by ffinfo — APPROVED** [review]

**Pull request merged**